src/docs/development/platform-integration/apple-watch.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Adding an Apple Watch extension"
+description: "Learn how to add an Apple Watch target to a Flutter app"
+---
+
+While you cannot build an Apple Watch app with Flutter, it is possible to add a native Apple Watch extension to a Flutter app.
+
+## Step 1: Enable bitcode in Xcode
+Apple Watch targets require bitcode to be enabled, so follow the steps [here](https://github.com/flutter/flutter/wiki/Creating-an-iOS-Bitcode-enabled-app-(experimental)) to use bitcode in your app.
+
+## Step 2: Add an Apple Watch target
+In the menu, select File > New > Target. Once the dialog opens, select *watchOS* at the top and click *Watch App for iOS App*. Click *Next*, enter a product name, and select *Enter*.
```

Comment:
We use bold for menu items ( https://developers.google.com/style/ui-elements ) =>
In the menu, select **File > New > Target**. Once the dialog opens,
select **watchOS** at the top and click **Watch App for iOS App**.
Click **Next**, enter a product name, and select **Enter**.

(Also, your line length is too long. We generally break on phrases and use no more than 80 chars.)